indexing/metadata.md

```diff
@@ -18,44 +18,199 @@ Including the following tags and metadata in each of your pages will improve the
+- **Used In:** Query matching, term frequency scoring
+
+### Title
+
+We look for the following fields:
+```
+<title>
+<meta property="og:title" content="Example" />
+```
+- **Detail:** Unique title of the page. If you want to include the agency or section name, place that after the actual page title. We use the `og:title` property as the result title if it appears to be more substantive than the `<title>` tag. Note, Open Graph elements are used to display previews of your content in FaceBook and some other social media platforms.
+- **Data Gathering Logic:** We prefer whichever title field is longer between the two.
+- **Used In:** Query matching, term frequency scoring
+
+
+### Description
+
+We look for the following fields:
+```
+<meta name="description" content="Example" />
+<meta property="og:description" content="Example" />
+```
+- **Detail:** Your well crafted, plain language summary of the page content. This will often be used by search engines in place of a page snippet. Be sure to include the keywords you want the page to rank well for. Best to limit to 160 characters, so it will not be truncated. [Read more here](https://moz.com/learn/seo/meta-description). Note, Open Graph elements are used to display previews of your content in FaceBook and some other social media platforms.
+- **Used In:** Query matching, term frequency scoring
+
+### Tags & Keywords
+
+We look for the following fields:
+```
+<meta name="keywords" content="example, example 2, example 3" />
+<meta name="dcterms.keywords" content="example" />
+<meta name="dc.subject" content="example" />` 
+<meta name="dcterms.subject" content="example" />
+<meta name="article:tag" content="example" />
+<meta name="article:section" content="example" />
+```
+- **Detail:** While not often used by commercial search engines due to [keyword stuffing](https://support.google.com/webmasters/answer/66358?hl=en), Search.gov indexes your keywords, if you have added them. 
+- **Data Gathering Logic:** We pull all fields listed into one "Tags" field for searching and future facet filtering.
+- **Used In:** Query matching, term frequency scoring, faceted search (upcoming feature)
+
+
+### Created Date
+
+We look for the following fields:
+```
+<meta property="article:published_time" content="YYYY-MM-DD" />
+<meta name="dc.date" content="YYYY-MM-DD" />
+<meta name="dc.date.created" content="YYYY-MM-DD" />`  
+<meta name="dcterms.created" content="YYYY-MM-DD" />
+```
+- **Detail:** The date fields are listed in order of preference. We only accept one `created` date, and use this list to determine the fallback options. Exact time is optional; [read more here](https://en.wikipedia.org/wiki/ISO_8601).
+- **Used In:** Page freshness scoring, faceted search (upcoming feature)
+
+
+### Changed Date
+
+We look for the following fields:
+```
+<meta property="article:modified_time" content="YYYY-MM-DD" />
+<lastmod>
```

Comment:
I can dig into this more, but I don't see evidence in the code of `<lastmod>` coming into play here.  The options are either `article:modified_time` with a fallback to the created date set by the logic above.

Reply:
Ah interesting, I'll remove it for now, I agree that I'm not seeing logic where the lastmod affects the changed value at first glance. Leaving it commented out so we can add it back if we see it does impact that field.